pkg/eventprocessor/impl/eventprocessor_test.go

```diff
@@ -419,6 +422,7 @@ func (acl *aclMock) CheckPrivileges(
 	tx *sql.Tx,
 	controller common.Address,
 	id tableland.TableID,
-	op tableland.Operation) (bool, error) {
+	op tableland.Operation,
+) (bool, error) {
```

Comment:
`gofumpt` strikes again. I like this too, this is actually how I would normally format this manually.